apps/SageAccountWeb/.env.localhost

```diff
@@ -0,0 +1,3 @@
+VITE_SYNAPSE_REPO_ENDPOINT=http://localhost:8080/services-repository-develop-SNAPSHOT
+VITE_SYNAPSE_PORTAL_ENDPOINT=http://localhost:8888
+
```

Comment:
Allow starting SageAccountWeb with `--mode=localhost` to point to local instance of Synapse backend